src/openrct2/platform/Platform2.cpp

```diff
@@ -24,4 +41,211 @@ namespace Platform
     {
         return platform_get_ticks();
     }
+
+    std::string GetEnvironmentVariable(const std::string &name)
+    {
+#ifdef _WIN32
```

Comment:
Why so many ifdefs? Don't we have the platform-specific files for that?

Reply:
I was going to let it grow a bit before I split into probably Platform.Windows.cpp, Platform.Linux.cpp etc. A class would be good too.